docs/VLSI/Sky130-Commercial-Tutorial.rst

```diff
@@ -129,9 +161,9 @@ To run DRC & LVS, and view the results in Calibre:
 .. code-block:: shell
 
     make drc tutorial=sky130-commercial
-    ./build/chipyard.TestHarness.TinyRocketConfig-ChipTop/drc-rundir/generated-scripts/view_drc
+    ./build/drc-rundir/generated-scripts/view_drc
     make lvs tutorial=sky130-commercial
-    ./build/chipyard.TestHarness.TinyRocketConfig-ChipTop/lvs-rundir/generated-scripts/view_lvs
+    ./build/lvs-rundir/generated-scripts/view_lvs
```

Comment:
Odd, the build dir shouldn't have changed...

Reply:
I thought it looked a bit verbose so I copied the [path specified in the ASAP7 tutorial](https://chipyard.readthedocs.io/en/stable/VLSI/ASAP7-Tutorial.html#drc-lvs).

Reply:
Oh, awkward. That's definitely wrong, since the build dir was made longer some releases ago of Chipyard. It's set [here](https://github.com/ucb-bar/chipyard/blob/a98a3163a800992ce408903f19536e01c0b61e95/vlsi/Makefile#L48-L51)
It would be wrong not to specify the correct path (people would get confused). Can you fix the ASAP7 tutorial docs too?

Reply:
Oh I see, thought this was done intentionally. Will fix in all the docs.